soc/arm/st_stm32/stm32wb/dts_fixup.h

```diff
@@ -155,6 +155,6 @@
 #define DT_ADC_1_CLOCK_BITS		DT_ST_STM32_ADC_50040000_CLOCK_BITS_0
 #define DT_ADC_1_CLOCK_BUS		DT_ST_STM32_ADC_50040000_CLOCK_BUS_0
 
-#define DT_WDT_0_NAME         DT_INST_0_ST_STM32_WATCHDOG_LABEL
+#define DT_WDT_0_NAME			DT_INST_0_ST_STM32_WATCHDOG_LABEL
```

Comment:
Well, move this fix in previous commit and we're good

Reply:
Addressed, thanks for catching that!

Reply:
Addressed, thanks!